lib/Importer/ModelLoader.cpp

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Importer/ModelLoader.h"
+
+#include <cassert>
+#include <string>
+
+using namespace glow;
```

Comment:
you do not need to `using namespace glow;` everything is already in namespace glow { below.

Reply:
ah missed that, why is it (namespace) closed right away, ModelLoader::getTensorByName and the rest are in glow namespace.

Reply:
I changed namespace closing point and removed `using`